core/templates/dev/head/pages/exploration-player-page/services/exploration-engine.service.ts

```diff
@@ -444,6 +444,13 @@ oppia.factory('ExplorationEngineService', [
       },
       isAnswerBeingProcessed: function() {
         return answerIsBeingProcessed;
+      },
+      currentStateName: function() {
```

Comment:
There must be a method/service which provides this data, have you checked?

Reply:
I have checked, but there is some cause of error in my method (for extracting image).
So i have to create one which fulfills my demand. Hope you remember i told you in the meeting.
I can create a small doc, which tells about the issue and how I overcome this. Is that fine ?